humanoid_league_team_communication/CMakeLists.txt

```diff
@@ -1,43 +1,40 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(humanoid_league_team_communication)
 
+INCLUDE(FindProtobuf)
+find_package(Protobuf REQUIRED)
+
 find_package(catkin REQUIRED COMPONENTS
-    roscpp
-    std_msgs
-    humanoid_league_msgs
-    geometry_msgs
-    bitbots_docs
+        roscpp
+        tf2
+        tf2_ros
+        tf2_geometry_msgs
+        std_msgs
+        humanoid_league_msgs
+        geometry_msgs
+        bitbots_docs
 )
-
+find_package(Boost REQUIRED COMPONENTS)
 catkin_package(
-    INCLUDE_DIRS include
-    LIBRARIES ${PROJECT_NAME}_lib
+        INCLUDE_DIRS include
+        LIBRARIES ${PROJECT_NAME}_lib
 )
+ADD_SUBDIRECTORY(../../lib/RobocupProtocol ${CMAKE_CURRENT_BINARY_DIR}/../RobocupProtocol)
```

Comment:
I would prefer to include the repository as an actual subdirectory, see [my comment on the other PR](https://github.com/bit-bots/RobocupProtocol/pull/1#pullrequestreview-636847455).